tests/unit/test_bin_load_sample_csv.py

```diff
@@ -19,7 +19,17 @@ def test_data_dir(request):
 @pytest.mark.parametrize("test_data_dir", ["good_csvs"], indirect=True)
 def test_create(connection, dbsession, settings, test_data_dir):
     """Most straightforward load works"""
-    ret = main(connection, settings, ["-d", test_data_dir])
+    ret = main(
+        connection,
+        settings,
+        [
+            "-d",
+            test_data_dir,
+            "--dev",
+            "--duplicates",
+            f"{test_data_dir}/duplicates.txt",
```

Comment:
Nit: Can we put some data in the test duplicates, to get some coverage on the duplicate processor, even if the duplicate data isn't used in tests 

*I can't comment on the empty files, so I'm putting it here*

Reply:
Tracked in #148